java-iam/.repo-metadata.json

```diff
@@ -5,7 +5,7 @@
   "api_description": "Manages access control for Google Cloud Platform resources",
   "client_documentation": "https://cloud.google.com/java/docs/reference/proto-google-iam-v1/latest/overview",
   "release_level": "stable",
-  "transport": "grpc",
+  "transport": "both",
```

Comment:
Do we know why the transport changed? Is it because this PR https://github.com/googleapis/sdk-platform-java/pull/2890?

Reply:
Yes, now `iam/v2` is used to get the transport.